include/revng/Model/RegisterState.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+/* MODEL-YAML
+name: RegisterState
+type: enum
+members:
+  - name: "No"
+  - name: NoOrDead
+  - name: Dead
+  - name: "Yes"
```

Comment:
`No` and `Yes` being booleans is the dumbest thing in the world.

Reply:
Agree :(